### PR TITLE
Account creation page and login page

### PR DESCRIPTION
**Pull request by JME469**
Structure for the login and creation page, also added basic JavaScript validation for the creation page (/public/js/signup.js)

### CODE REVIEW

File: public/pages/feed.html
Comment (KaruzG) [inline public/pages/feed.html:1]: This is a PR for the login not the feed

**Review by KaruzG — CHANGES_REQUESTED** [review]
I think you unintencionally added commits from another feature (feed)

**Review by JME469 — COMMENTED** [review]
I already separated the unwanted commits (feed page changes) into another branch, also solved conflicts

**Review by KaruzG — APPROVED** [review]
Nice! 💥💥💥

**Review by Rafval18 — APPROVED** [review]
Good Job!

**Pull request merged**